CardBox/Views/PositionRequestView.swift

```diff
@@ -7,10 +7,19 @@
 
 import SwiftUI
 
-struct DeckPositionView: View {
+struct PositionRequestView: View {
```

Comment:
do we need to restrict the position here based on the number of cards in the hand/deck? like how far it can go?

Reply:
Yes